partiql-planner/src/main/kotlin/org/partiql/planner/internal/casts/CastTable.kt

```diff
@@ -63,6 +63,11 @@ internal class CastTable private constructor(
                     cast(it)
                 }
             }
+            graph[PType.VARIANT] = relationships {
+                PType.codes().map {
+                    cast(it)
+                }
+            }
```

Comment:
Interesting, may be worth a one-line note like "a variant can be cast to any type"